web/src/app/services/ServiceLabelList.js

```diff
@@ -38,29 +38,16 @@ const sortItems = (a, b) => {
   return 0
 }
 
-@withStyles(styles)
-export default class ServiceLabelList extends React.PureComponent {
-  static propTypes = {
-    serviceID: p.string.isRequired,
-  }
+const useStyles = makeStyles(styles)
 
-  state = {
-    create: false,
-    editKey: null,
-    deleteKey: null,
-  }
+export default function ServiceLabelList(props) {
+  const { serviceID } = props
```

Comment:
nit/preference, since we aren't using `props` anywhere else in this component:
```suggestion
export default function ServiceLabelList({ serviceID }) {
```